components/button-cards/contact-card/contact-card.js

```diff
@@ -0,0 +1,20 @@
+import { setCardContentMaximumHeight, resetCardContentHeights } from 'components/button-cards/card.js'
+
+window.addEventListener('load', function() {
+  setCardContentMaximumHeight('.card-row', '.msds-contact-card__headline')
+  setCardContentMaximumHeight('.card-row', '.msds-contact-card__body-contact-person')
+  setCardContentMaximumHeight('.card-row', '.msds-contact-card__body-contact-position')
+  setCardContentMaximumHeight('.card-row', '.msds-contact-card__body-contact-details-wrapper')
+})
+
+window.onresize = function() {
```

Comment:
I think we need to build a throttle on this, ive read that resize window events can be heavy on the perfomance if not handled well

Reply:
Oh yeah they can be and as we put more elements into the HTML it causes a lot more bloat on the clients devices. So yeah it would be a good idea to do an investigation on this